app/main.go

```diff
@@ -32,20 +32,19 @@ func main() {
 		return
 	}
 
+	defer log.Print("bye!")
```

Comment:
Would be nice to make it clear this is the app exiting: logs are mixed with probe when running under docker.